src/components/widgets/commit/Commit.tsx

```diff
@@ -32,10 +32,8 @@ export function Commit() {
   if (!productUuid && sellerId) {
     return <p>Missing 'productUuid' query param</p>;
   }
-  const lookAndFeel = searchParams.get("lookAndFeel") as "regular" | "modal";
-  if (!lookAndFeel) {
-    return <p>Missing 'lookAndFeel' query param</p>;
-  }
+  const lookAndFeel =
```

Comment:
simplify the usage, making the lookAndFeel attribute optional for the integrator